lib/deployment_scheduler.rb

```diff
@@ -4,15 +4,25 @@ def initialize(definition_parser = DeploymentDefinitionParser.new)
   end
 
   def schedule
-    Deployment.all.each do |deployment|
-      begin
-        process(deployment)
-        deployment.update!(last_error_msg: nil, last_error_at: nil)
-      rescue Exception => e
-        deployment.update!(last_error_msg: "#{e.class.name}: #{e.message}", last_error_at: Time.now)
-        Rails.logger.warn "#{e.class.name}: #{e.message}}"
+    deployment_count = 0
+
+    elapsed_time = Benchmark.realtime do
+      Deployment.find_each do |deployment|
+        begin
+          process(deployment)
+          if deployment.last_error_msg != nil or deployment.last_error_at != nil
+            deployment.update!(last_error_msg: nil, last_error_at: nil)
+          end
+        rescue Exception => e
+          deployment.update!(last_error_msg: "#{e.class.name}: #{e.message}", last_error_at: Time.now)
+          p "#{deployment.name}: Error: #{e.class.name}: #{e.message}"
```

Comment:
better using `p` or `Rails.logger` @nieltg ?

Reply:
We have tried to use `Rails.logger` but it's not showing on the Docker environment.
@giosakti 

I'll try again to inspect why it's not working.

Reply:
now that you found the root cause, can you revert this back to logger?

Reply:
ping @nieltg

Reply:
Sorry, I just read this one. I prefer for the other PR to be merged first, so I can rebase this one later. Now, since most PR has been merged, I'll handle this one again.